features/machine/machine.feature

```diff
@@ -332,12 +332,12 @@ Feature: Machine features testing
     And I switch to cluster admin pseudo user
     And I use the "openshift-machine-api" project
     Then admin ensures machine number is restored after scenario
-
-    Given I store the last provisioned machine in the :machine clipboard
-    When evaluation of `machine(cb.machine).aws_ami_id` is stored in the :default_ami_id clipboard
-    And evaluation of `machine(cb.machine).aws_availability_zone` is stored in the :default_availability_zone clipboard
-    And evaluation of `machine(cb.machine).aws_subnet` is stored in the :default_subnet clipboard
-    And evaluation of `machine(cb.machine).aws_iamInstanceProfile` is stored in the :default_iamInstanceProfile clipboard
+    
+    Given I pick a earliest created machineset and store in :machineset clipboard
```

Comment:
@miyadav can you tell me why we need this step? Is there any difference between getting value from machineset and machine?

Reply:
If we have a cluster with windows machines we need to make use of this, in that case otherwise we have to capture the value from master machine, which I thought of avoiding as mapi doesnt have anything to do with master machines.